fedimint-bitcoind/Cargo.toml

```diff
@@ -26,6 +26,11 @@ bitcoincore-rpc = { workspace = true, optional = true }
 esplora-client = { workspace = true, optional = true }
 fedimint-core = { workspace = true }
 fedimint-logging = { workspace = true }
+futures = { workspace = true }
 hex = { workspace = true }
+jaq-core = { workspace = true }
```

Comment:
I wonder if we should move it to some `fedimint-bitcoind-feerate`, so that other `fedimint-bitcoind` depedents don't need this extra stuff now, but good enough I guess.